wurst/objects/abilities/beastmaster/Druid/BarkSkin.wurst

```diff
@@ -0,0 +1,89 @@
+package BarkSkin
+
+// Standard Library Imports:
+import Icons
+import ChannelAbilityPreset
+import ClosureEvents
+import ClosureTimers
+import BuffObjEditing
+import BuffIds
+import Abilities
+import DamageEvent
+
+// Local Imports:
+import LocalObjectIDs
+import ToolTipsUtils
+
+
+let COOLDOWN = 45.
+let MANA_COST = 15
+let ARMOR_BONUS = 4
+let THORNS_DMG = .15
+let DURATION = 15.
+let CAST_RANGE = 15.
+let BUTTON_POS_X = 3
+let BUTTON_POS_Y = 2
+let HOTKEY = "R"
+let NAME = "Bark Skin"
+let ICON = Icons.bTNThornShield
+let TOOLTIP_BASIC = "Bark Skin"
+let TOOLTIP_EXT   = "Imbue your target with the strength of the forest, giving it " + ARMOR_BONUS.toReal().toToolTipGreen()
```

Comment:
Maybe use "nature" instead of "forest" in the tooltip, we're on an island context and most of the animals are named with the "jungle" prefix, forest feels out of place for me. More over you're using the word "nature" in the buff tooltip.

The tooltip doesn't mention damage returned?